devices/bosch.js

```diff
@@ -139,9 +140,12 @@ const fzLocal = {
                 result.boost = (Object.keys(stateOffOn)[data[0x4043]]);
             }
             if (data.hasOwnProperty(0x4007)) {
-                const opModes = {0: 'auto', 1: 'heat', 2: 'unknowm 2', 3: 'unknonw 3', 4: 'unknown 4', 5: 'off'};
+                const opModes = {0: 'auto', 1: 'heat', 2: 'unknown 2', 3: 'unknown 3', 4: 'unknown 4', 5: 'off'};
```

Comment:
Can you make these snake case? e.g `unknown 2` -> `unknown_2`

Reply:
Sure. Changed it..